test/unit/parameters/common_parameter_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class CommonParameterTest < ActiveSupport::TestCase
+  should validate_presence_of(:name)
```

Comment:
It would be useful to have these tests in all Parameter subclasses, can they go into ParameterValidations or similar?